proto/jobworker/v1/service_api.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto3";
+
+package jobworker.v1;
+
+option go_package = "github.com/tjper/teleport/proto/jobworker/v1";
+
+// JobWorkerService provides methods that allow clients to start and interact
+// with arbitrary commands running on the service's host.
+service JobWorkerService {
+  rpc Start(StartRequest) returns (StartResponse){}
+  rpc Stop(StopRequest) returns (StopResponse){}
+  rpc Status(StatusRequest) returns (StatusResponse){}
+  rpc Output(OutputRequest) returns (stream OutputResponse){}
+}
+
+// StartRequest specifies job details for JobWorkerService.Start.
+message StartRequest {
+  // command is the arbitrary command to execute on JobWorkerService's host.
+  string command  = 1;
+  // limits are the resource limits to enforce on the job.
+  Limits limits   = 2;
+}
+
+// StartResponse informs clients started job details.
+message StartResponse {
+  string job_id  = 1;
+  // command is the arbitrary command that has been executed.
+  string command = 2;
+  // status is the human-readable current status of the job.
```

Comment:
What exactly is a human-readable status?

Reply:
Answered below in question regarding StatusResponse.status.